gnucash/register/register-gnome/combocell-gnome.c

```diff
@@ -944,6 +944,10 @@ popup_get_height (G_GNUC_UNUSED GtkWidget* widget,
         height = height + minh;
     }
 
+    // if popup_allocation_height set use that
+    if (box->item_edit->popup_allocation_height != -1)
+        height = box->item_edit->popup_allocation_height;
+
```

Comment:
ISTM this should go first with the `count * cell_height` as an `else` fallback. Is this partly to figure out how big a window to ask for or otherwise set `height` before the popup is realized and has an allocation?